languages/java/exercises/concept/constructors/.docs/hints.md

```diff
@@ -0,0 +1,69 @@
+Creating an instance of a _class_ is done by calling its [_constructor_][constructors] through the [`new` operator][new]. A constructor is a special type of method whose goal is to initialize a newly created instance. Constructors look like regular methods, but without a return type and with a name that matches the classes' name.
```

Comment:
I assume the hints file is still to be worked on, as this looks like a copy of the introduction :) For a description of what this file should look like, see [these docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docshintsmd). The Java arrays exercise has a nice [hints file](https://github.com/exercism/v3/blob/master/languages/java/exercises/concept/arrays/.docs/hints.md) that you can use to see what a hints file should look like.

Reply:
Good catch. Must have copied the wrong file.

Reply:
Copied the hints file from C# and updated.